tempodb/encoding/vparquet5/create.go

```diff
@@ -292,6 +295,7 @@ func estimateMarshalledSizeFromTrace(tr *Trace) (size int) {
 	for _, rs := range tr.ResourceSpans {
 		size += estimateAttrSize(rs.Resource.Attrs)
 		size += 21 // 21 resource lvl fields including dedicated attributes
+		size += 10 // 10 dedicated columns
```

Comment:
Since we don't have the well known attributes anymore, the 21 value is not accurate anymore for this schema right? Are there 10 dedicated columns per scopes now or 15 string + int?